kubernetes/kubernetes.go

```diff
@@ -0,0 +1,56 @@
+package kubernetes
```

Comment:
Duplicate of https://github.com/giantswarm/kubeconfig/pull/20

"Vendored" my changes to that package in this file, because in order to update it to the latest version, some modules have to be upgraded, which are incompatible currently (see microerror). This is a temporary solution